x-pack/plugins/ml/public/application/components/job_selector/use_job_selection.ts

```diff
@@ -71,23 +73,21 @@ export const useJobSelection = (jobs: MlJobWithTimeRange[]) => {
   }, [invalidIds]);
 
   useEffect(() => {
-    // if there are no valid ids, warn and then select the first job
+    // if there are no valid ids, ask the user to provide job selection with the flyout
     if (validIds.length === 0 && jobs.length > 0) {
-      toastNotifications.addWarning(
-        i18n.translate('xpack.ml.jobSelect.noJobsSelectedWarningMessage', {
-          defaultMessage: 'No jobs selected, auto selecting first job',
+      getJobSelection({ singleSelection: false })
```

Comment:
When opening the Single Metric Viewer, all the radio buttons are enabled in the flyout, even for jobs which cannot be viewed in the Single Metric Viewer. Only happens when opening the view from the side nav.

Reply:
Fixed in https://github.com/elastic/kibana/pull/123805/commits/ce827e95aad6dcd0effdce8850b135e120d19755